packages/web/src/components/search/CategorySearch.js

```diff
@@ -858,32 +859,25 @@ class CategorySearch extends Component {
 		}
 
 		if (this.state.currentValue && this.state.suggestions.length && filteredCategories.length) {
-			let categorySuggestions = [
-				{
-					label: `${this.state.currentValue} in all categories`,
-					value: this.state.currentValue,
-					category: '*',
-					// no source object exists for category based suggestions
-					source: null,
-				},
-				{
-					label: `${this.state.currentValue} in ${filteredCategories[0].key}`,
+			const categorySuggestions = [];
```

Comment:
How about slicing the array of `filteredCategories` and then mapping over it. Something like this: 

```
const size = filteredCategories.length < categorySuggestionsListSize ?  filteredCategories.length  : categorySuggestionsListSize;
const categoriesToDisplay = filteredCatogories.slice(0, size).map((item) => {
   // push to array ( try using spread operator if possible )
});
```

Reply:
Also please try to test it locally rigorously.